docs/docs-env.yml

```
@@ -0,0 +1,18 @@
+name: mBuild-docs
+channels:
+  - conda-forge
+  - omnia
+  - mosdef
+dependencies:
+  - python=3.7
+  - openmm
+  - parmed
+  - pip:
+    - sphinx
+    - sphinx_rtd_theme
+    - sphinxcontrib-svg2pdfconverter
+    - nbsphinx
+    - networkx >=2.0
+    - lark-parser
+    - requests
+    - lxml
```

can these packages be installed with conda?
```suggestion
  - sphinx
  - sphinx_rtd_theme
  - sphinxcontrib-svg2pdfconverter
  - nbsphinx
  - networkx >=2.0
  - lark-parser
  - requests
  - lxml
```